Iterable/iterable-web-sdk
# [Part 1] In-App Cypress Tests - Raw Data

## JIRA Ticket(s) if any

* [MOB-3928](https://iterable.atlassian.net/browse/MOB-3928)

## Description

Adds in-app cypress tests for requesting raw data and painting it to the screen

🚨  Tests around auto-displaying in-app messages coming in the next PR

## Test Steps

1. Run `yarn install:all && yarn start:all:react` to start app
2. Run `cd react-example && yarn cypress` to start the Cypress runner
3. Run all Cypress tests and ensure suite passes
